lib/foods.js

```diff
@@ -1,28 +1,56 @@
+const local = 'http://localhost:3000/api/foods'
```

Comment:
@TheKaterTot I'll remove this once all of the AJAX calls to the API have been completed! I prefer to test things out locally vs. in production.